docs/decisions/0006-ledger-balance-transaction-states.rst

```diff
@@ -0,0 +1,37 @@
+0006 Ledger balance and transaction states
+##########################################
+
+Status
+******
+
+**Accepted** (May 2023)
+
+Context
+*******
+
+We will take a strong position on the states of transactions that should count when
+computing the balance of a ledger.
```

Comment:
nit: explain _why_ we're taking a strong position on the states of transactions? Useful context before reading about the decision.